src/docs/sdk/performance/time-to-initial-full-display.mdx

```diff
@@ -47,6 +48,12 @@ Scenario: Option enabled
     When the SDK creates a new screen load transaction
     Then the Full Display span is started
 
+Scenario: App started
+    Given the enableTimeToFullDisplay option is enabled
+    When the app starts
+    And the SDK start the span
```

Comment:
Should the SDK also change the recorded measurements? If so, please add it to this scenario.

Reply:
The measurement is always the span duration. If the timestamp is changed, the span duration changes, so the SDK shouldn't do anything special regarding it.
At least this is how i see it. Would it make more sense to explicitly list it in the scenarios? (including all the other scenarios when the SDK modifies the start or end timestamp of the span)

Reply:
Yes, I would explicitly list that in this scenario.

Reply:
@philipphofmann is the wording fine?